CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # cuML 0.15.0 (Date TBD)
 
 ## New Features
+- PR #2580: Added model persistence via joblib in each section of estimator_intro.ipynb
```

Comment:
```suggestion
- PR #2581: Added model persistence via joblib in each section of estimator_intro.ipynb
```
PR number is wrong and that's why CI failed. Besides that, the PR looks good once conflicts are solved